packages/server/tests/acceptance/htsPrecompile.spec.ts

```diff
@@ -586,6 +586,65 @@ describe('HTS Precompile Acceptance Tests', async function () {
     });
   });
 
+  // Depends on https://github.com/hashgraph/hedera-services/pull/3756
+  xdescribe('HTS Precompile Token Expiry Info Tests', async function() {
+    it('should be able to get fungible token expiry info', async function() {
+      const getTokenExpiryInfoTx = (await baseHTSContract.getTokenExpiryInfoPublic(HTSTokenContractAddress));
+      const responseCode = (await getTokenExpiryInfoTx.wait()).events.filter(e => e.event === 'ResponseCode')[0].args.responseCode;
+      const tokenExpiryInfo = (await getTokenExpiryInfoTx.wait()).events.filter(e => e.event === 'TokenExpiryInfo')[0].args.expiryInfo;
+
+      expect(responseCode).to.equal(TX_SUCCESS_CODE);
+      expect(tokenExpiryInfo.autoRenewPeriod).to.equal(7776000);
```

Comment:
nit: add check for `second` also here and below

Reply:
Done